app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,14 @@
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    # @user = User.includes(:posts).where(user: {id: params[:user_id]}))
+    @user = User.includes(:posts).find(params[:user_id])
```

Comment:
- [ ] Please remove the above-commented line of code in order to keep your codebase clean 😃 👍 It is also a best practice 💡